nodejs/clients.js

```diff
@@ -1,23 +1,22 @@
-const { readMasterKey, ClientBuilder } = require("./helpers")
+const { readMasterKey, CsfleHelper } = require("./helpers")
 
 const localMasterKey = readMasterKey()
 
-const clientBuilder = new ClientBuilder({
+const csfleHelper = new CsfleHelper({
   kmsProviders: {
     local: {
-      key: localMasterKey
-    }
-  }
+      key: localMasterKey,
+    },
+  },
 })
 // Wrapping everything in an async iife to allow use of async/await
 ;(async () => {
```

Comment:
I prefer the following pattern:
```
async function main() {

}

main().catch(console.dir);
```
so a user isn't presented with a "Uncaught Promise rejection" error if something goes wrong